src/patternfly/components/DescriptionList/examples/DescriptionList.md

```diff
@@ -117,6 +117,33 @@ Column fill will modify the default placement of description list groups to fill
 {{> description-list__example description-list--title="Default inline grid" description-list--modifier="pf-m-3-col pf-m-inline-grid"}}
 ```
 
+## Display and card variants
+
+### Display variant
+```hbs
+{{> description-list__example description-list--modifier="pf-m-display-lg pf-m-2-col-on-lg" description-list--title="Card DL"}}
+```
+
+### Group as card
```

Comment:
What do you mean by "group"?

Reply:
Our description list `<dd>` and `<dt>` examples are wrapped by `.pf-c-description-list__group`. Do you think I should change the verbiage?